exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -129,6 +131,24 @@ fun <T> allFrom(array: Array<T>): Op<T> = AllAnyFromArrayOp(false, array)
 /** Returns this table wrapped in the `ALL` operator. This function is only supported by MySQL, PostgreSQL, and H2 dialects. */
 fun <T> allFrom(table: Table): Op<T> = AllAnyFromTableOp(false, table)
 
+/**
+ * Returns the array element stored at the one-based [index] position, or `null` if the stored array itself is null.
+ *
+ * @sample org.jetbrains.exposed.sql.tests.shared.types.ArrayColumnTypeTests.testSelectUsingArrayElementAt
+ */
+fun <E, T : List<E>?> ExpressionWithColumnType<T>.elementAt(index: Int): ArrayElementAt<E, T> =
```

Comment:
Please consider adding a function for `column[index]` access

Reply:
@e5l Done! Is there any value in keeping `elementAt()` now that there is the `get()` operator for index access?

Reply:
yep, we can drop it